controllers/topics-controller.js

```diff
@@ -0,0 +1,7 @@
+const { selectTopics } = require("../models/models-controller");
```

Comment:
Just be careful with your model filename here. It should be topics-model, rather than models-controller 🙂